lib/mutant/subject/method.rb

```diff
@@ -13,12 +13,12 @@ class Method < self
 
       # Return method name
       #
-      # @return [Symbol]
+      # @return [Expression]
       #
       # @api private
       #
       def name
-        node.children[self.class::NAME_INDEX]
+        node.children.at(self.class::NAME_INDEX)
```

Comment:
There is no such a mutation. Easy to add, please open a ticket. But I think we actually should use `Array#fetch` here. As as `#at` defaults to `nil` and we know the name is always present.

Ideally we mutate `#[]` to `#fetch` and `#at`.

Also this change is not related at all to the subject of this PR, I'll move it to another iteration.